### ISSUE TITLE
Undefined constant referenced in function parameter default value is not detected

### ISSUE THREAD

**Issue by weirdan** [vimeo/psalm#512]
```php
class A {
   public function doSomething(
          int $howManyTimes = self::DEFAULT_TIMES // should complain about undefined constant
   ): void {}
}
```

**Comment by muglug** [comment]
Thanks! This also led me to find a bug in another test.
